javascript/src/main/components/Footer/AppFooter.js

```diff
@@ -3,7 +3,7 @@ import React from "react";
 const AppFooter = () => {
   return (
     <footer className="bg-light p-3 text-center">
-      This is a demo of a minimal React frontend + Spring Boot backend app.
+      This is abcd demo of a minimal React frontend + Spring Boot backend app.
```

Comment:
I don't understand why you changed the footer here. It might've been a mistake

Reply:
Just changed it back with my most recent commit.